projects/go-style-guide/src/app/features/ui-kit/components/accordion-docs/components/accordion-panel-docs/accordion-panel-docs.component.html

```diff
@@ -214,11 +214,7 @@ <h4 class="go-heading-4 go-heading--underlined">View</h4>
           </go-accordion-panel>
           <go-accordion-panel heading="Projected Heading">
             <ng-template #headerContent>
-              <div class="go-container go-container--justify-end">
-                <div class="go-column go-column--no-padding">
-                  <go-button>Projected</go-button>
-                </div>
-              </div>
+              <go-pill [removable]="false">Projected</go-pill>
```

Comment:
Should we update the `headerTemplateExample` code snippet referenced below with this change?

Reply:
Yes we should, good catch. I updated that.